src/chains.jl

```diff
@@ -192,14 +192,15 @@ names as the values.
 Passing `flatten=true` will return a `NamedTuple` with keys ungrouped.
 
 # Example
-```julia
-julia> chn = Chains([1:5 6:10]);
+
+```jldoctest
+julia> chn = Chains([1:2 3:4]);
 
 julia> get(chn, :param_1)
-(param_1 = [1; 2; … ; 4; 5],)
+(param_1 = [1; 2],)
 
-julia> get(chn, [:param_1, :param_2])
-(param_2 = [6; 7; … ; 9; 10], param_1 = [1; 2; … ; 4; 5])
+julia> get(chn, [:param_2])
+(param_2 = [3; 4],)
```

Comment:
I guess this part is how you're dealing with the displaysize-issue you mentioned in the comment?

Reply:
Yep, I just reduced the output such that it should work even with tiny displays.